adserver/api/serializers.py

```diff
@@ -28,7 +29,13 @@ class AdPlacementSerializer(serializers.Serializer):
     div_id = serializers.CharField()
     ad_type = serializers.CharField(required=True)
     priority = serializers.IntegerField(
-        default=1, min_value=1, max_value=10, required=False
+        default=1,
+        min_value=1,
+        max_value=10,
+        required=False,
+        help_text=_(
```

Comment:
help_text is useful! 💯 Though I don't think it's shown anywhere, at least useful internally 👍